modules/acm/src/main/java/edu/gemini/epics/acm/CaObserveSenderImpl.java

```diff
@@ -588,7 +570,13 @@ public State onTimeout() {
         final short stopMark;
         final short abortMark;
 
-        WaitObserveCompletion(CaCommandMonitorImpl cm, short stopMark, short abortMark) {
+        WaitObserveCompletion(CaCommandMonitorImpl cm) {
+            this.cm = cm;
+            this.stopMark = MRK_IDLE;
+            this.abortMark = MRK_IDLE;
```

Comment:
Assume initial value for stop.MARK and abort.MARK channels. This still works, because we only care about them when they change while the observation is in progress (i.e. while in state `WaitObserveCompletion`)